src/transformers/modeling_lxmert.py

```diff
@@ -473,27 +575,27 @@ def forward(
         for layer_module in self.layer:
             l_outputs = layer_module(lang_feats, visn_attention_mask, output_attentions=output_attentions)
             lang_feats = l_outputs[0]
-            if all_l_attentions:
+            all_l_hidden_states = all_l_hidden_states + (lang_feats,)
+            if all_l_attentions is not None:
                 all_l_attentions = all_l_attentions + (l_outputs[1],)
 
         # Run relational layers
         for layer_module in self.r_layers:
             v_outputs = layer_module(visn_feats, visn_attention_mask, output_attentions=output_attentions)
             visn_feats = v_outputs[0]
-            if all_v_attentions:
```

Comment:
`bool(()) == False`, need to enforce that it is not `None`.

Reply:
Ah so that is why it was "None" before, makes a lot of sense. I can definetely fix this